baseband/asp/tests/test_asp.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+
+from ... import asp
+from ...data import SAMPLE_ASP as SAMPLE_FILE
+
+
+__all__ = ['TestASP']
+
+
+class TestASP(object):
+    def __init__(self):
+        self.frame = None
+
+    def setup(self):
+        with open(SAMPLE_FILE, 'rb') as fh:
+            sr = asp.ASPStreamReader(fh)
+            self.frame = sr.read_frame()
+            self.header = self.frame.header
+            self.payload = self.frame.payload
+            assert fh.tell() == self.header.nbytes + self.header.file_header.nbytes + self.payload.nbytes, \
+                "TestASP::setup: file position not expected"
```

Comment:
Not necessary to have "TestASP::setup:" - the test suite will already report which test returns an error.  Just raise `"file position not expected."`